src/components/SubscriptionBanner.js

```diff
@@ -31,26 +31,25 @@ const SubscriptionBanner = ({handleSubmit, subscribed}) => {
   }
 }
 
-
-
-
 SubscriptionBanner.propTypes = {
   handleSubmit: PropTypes.func.isRequired,
-  subscribed: PropTypes.bool.isRequired
+  subscribed: PropTypes.bool.isRequired,
 }
 
 const mapStateToProps = (state) => {
   return {
-    subscribed: state.subscribed
+    subscribed: state.subscribed,
+    email: state.email
   };
 };
 
 const mapDispatchToProps = (dispatch) => {
   return {
     handleSubmit: (e) => {
       e.preventDefault()
-      dispatch(setSubscribed(true))
-      // TODO: persist to DB with API call
+      // TODO: this is hacky af
+      const email = e.target.children[0].value
```

Comment:
Is there a better way to handle this? I could not use refs because the components were stateless.